Source/Controllers/SubviewControllers/SPUserManager.m

```diff
@@ -437,7 +437,7 @@ - (void)_initializeSchemaPrivsForChild:(SPUserMO *)child fromData:(NSArray *)dat
 		SPPrivilegesMO *dbPriv = [NSEntityDescription insertNewObjectForEntityForName:@"Privileges" inManagedObjectContext:[self managedObjectContext]];
 
         // some error checks
-        if (rowDict[@"Db"] && doneRecordError == NO){
+        if (rowDict[@"DB"] && doneRecordError == NO){
```

Comment:
What does this change do exactly? Can we be sure that we always get DB instead of Db, or should we check for both?

Reply:
the Core Data model has the key `Db`, so it was crashing if the server returned `DB`. I wanted to log which servers were sending `DB`, but I mixed it up, so we got loads of errors. Further down in the code, if the server sent `DB`, I switch it to `Db` to prevent the crash. This was just for investigation really.